packages/components/src/date-time/types.ts

```diff
@@ -21,15 +21,11 @@ export type UpdateOnBlurAsIntegerFieldProps = {
 	children?: ReactNode;
 };
 
-export type DateTimeString = string;
-
-export type ValidDateTimeInput = Date | string | number | null;
```

Comment:
@ciampo was right that this doesn't work so well with stories. I mean, it works, but it's not very useful for someone reading the story as documentation to just see `ValidDateTimeInput`.